### ISSUE TITLE
Add support for environment variables (incl. secrets)

### ISSUE THREAD

**Issue by dgzlopes** [grafana/k6-cloud-feature-requests#11]
Instead of passing my secrets as environment variables with the CLI all the time, I would love to store these in k6 Cloud and access them from all my scripts. Maybe these secrets can be set per organization, project, or account.

GitHub example:
![gitexample](https://user-images.githubusercontent.com/8228060/110775996-d9a67180-825f-11eb-8441-f63d3a472313.png)

**Comment by na--** [comment]
Given that this would almost certainly be implemented as [k6 environment variables](https://k6.io/docs/using-k6/environment-variables), it might be in a slight conflict with https://github.com/k6io/cloud-feature-requests/issues/9, since we probably don't want to display the values of secrets in the UI, but want to display the values of non-secret environment variables.

**Comment by dgzlopes** [comment]
Yup, probably implementing it with environment variables is the best! I think that secrets managed by k6 cloud can have some kind of prefix. It's not perfect, but it's a workaround for #9.

**Comment by robingustafsson** [comment]
Yes, this is something we plan to implement. There's an initial project doc for it. What we've discussed previously is that it would allow for env var collections (on a project level) where you could switch between different sets of env vars via the k6 CLI, something like `k6 cloud --env-collection "production" script.js`. Each env vars defined in a collection in the cloud should be able to be marked as secret and would be treated as such when displayed, eg. on the result page. Any env var defined using `k6 cloud -e NAME=VALUE` would override any matching env var from a collection. This is something that's been requested by customers.

**Comment by PkHutch** [comment]
TLDR: Bump

Adding to make a bit more noise as this would be pretty beneficial, in our particular use case it would be because it would be nice for users to run tests on k6 Cloud from their local without being directly exposed to the auth token necessary for testing an API.

The auth token can be passed in via the command line, but this requires the user to have local access to the token, which is what is I'm hoping to avoid.

I was thinking maybe it would be possible to play around with scenarios, but I'm not sure if storing a token in a scenario is a good idea.

**Comment by PaulMEdwards** [comment]
I inquired with Support asking whether/where I could find ENV var management in the Cloud UI and was directed to this issue and also told:

> The team is currently working on implementing this feature, and we are testing it on our side.  There's no concrete release date at the moment.

**Comment by markjmeier** [comment]
This has been implemented - apologies for being slow to update this issue.  Cloud Envs can be set within the k6 cloud: https://k6.io/docs/cloud/manage/environment-variables/#manage-environment-variables
